plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgresTaskFailureRecoveryTest.java

```diff
@@ -22,4 +25,38 @@ public TestPostgresTaskFailureRecoveryTest()
     {
         super(RetryPolicy.TASK);
     }
+
+    @Test
+    @Override
+    protected void testDeleteWithSubquery()
+    {
+        // TODO: support merge with fte
+        // The test code is identical to BaseFailureRecoveryTest#testDeleteWithSubquery
+        // The test can run successfully but the MERGE implement details is not guarantee fte, overriding to claim it
```

Comment:
I don't fully understand the comment. Can you please run the MERGE query here and assert that it fails? Or is the failure not deterministic? Same below.

Reply:
we would get `assertThatThrownBy(super::testDeleteWithSubquery).hasMessageContaining("Expecting code to raise a throwable.");`
It's because the test defined in BaseFailureRecoveryTest can run successfully with current implementation

Reply:
But the `BaseJdbcFailureRecoveryTest` overwrite it and expect the test fail

Reply:
I’ve copied the code from `BaseFailureRecoveryTest`. Which version do you think is better? Do you prefer keeping `the assertThatThrownBy(super::testDeleteWithSubquery).hasMessageContaining("Expecting code to raise a throwable.");` ?